sdks/python/apache_beam/io/gcp/big_query_query_to_table_it_test.py

```diff
@@ -0,0 +1,167 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+Integration test for Google Cloud BigQuery.
+"""
+# pylint: disable=wrong-import-order, wrong-import-position
+from __future__ import absolute_import
+
+import datetime
+import logging
+import unittest
+import time
+
+from hamcrest.core.core.allof import all_of
+from nose.plugins.attrib import attr
+
+from apache_beam.io.gcp import big_query_query_to_table_pipeline
+from apache_beam.io.gcp.bigquery import BigQueryWrapper
+from apache_beam.io.gcp.internal.clients import bigquery
+from apache_beam.testing import test_utils
+from apache_beam.io.gcp.tests.bigquery_matcher import BigqueryMatcher
+from apache_beam.testing.pipeline_verifiers import PipelineStateMatcher
+from apache_beam.testing.test_pipeline import TestPipeline
+
+BIG_QUERY_DATASET_ID = 'python_query_to_table_'
+NEW_TYPES_INPUT_TABLE = 'python_new_types_table'
+NEW_TYPES_OUTPUT_SCHEMA = (
+  '{"fields": [{"name": "bytes","type": "BYTES"},'
+  '{"name": "date","type": "DATE"},{"name": "time","type": "TIME"}]}')
+NEW_TYPES_OUTPUT_VERIFY_QUERY = ('SELECT date FROM [%s];')
```

Comment:
Ideally, this verify query should also query bytes and time. But there are some bugs within current bigquery client version we are using. For time, it cannot process .9999999: https://github.com/GoogleCloudPlatform/google-cloud-python/issues/5750. For bytes, it cannot query bytes type properly.